exp/controllers/awsmachinepool_controller.go

```diff
@@ -77,7 +77,6 @@ func (r *AWSMachinePoolReconciler) getEC2Service(scope *scope.ClusterScope) serv
 // +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups="",resources=secrets;,verbs=get;list;watch
 
-// Reconcile TODO: add comment bc exported
```

Comment:
We will actually need a comment here to pass the linter i think. 

`make verify` and `make lint` should help you out here.

Reply:
trying both make commands now

Reply:
Hahaha I actually added that comment there of my own accord a while back because my linter was yelling at me. 
I removed it because I thought I was just being picky because everyone else's reconcile() function didn't have a comment, so I was confused 😂 Will address it now.